plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -165,6 +162,7 @@
     public static final String OPENSHIFT_DEPLOYMENT_LABEL               = "deployment";
     public static final String CHE_MOUNTED_WORKSPACE_FOLDER             = "/workspace-logs";
     public static final String WORKSPACE_LOGS_FOLDER_SUFFIX             = "-logs";
+    public static final int OPENSHIFT_WAIT_POD_TIMEOUT                  = 240;
```

Comment:
@amisevsk good point, but I'm not sure about 5 seconds - in che starter[1] stop timeout is 150 ms  


https://github.com/redhat-developer/che-starter/blob/master/src/main/resources/application.properties#L21